apps/chat/src/store/prompts/prompts.epics.ts

```diff
@@ -785,11 +788,13 @@ const deleteChosenPromptsEpic: AppEpic = (action$, state$) =>
             folders: folders.filter(
               (folder) =>
                 !fullyChosenFolderIds.includes(`${folder.id}/`) &&
-                prompts.some((p) => p.id.startsWith(`${folder.id}/`)),
+                (prompts.some((p) => p.id.startsWith(`${folder.id}/`)) ||
+                  emptyFoldersIds.some((id) => id === folder.id)),
             ),
           }),
         ),
         of(PromptsActions.resetChosenPrompts()),
+        of(PromptsActions.resetChosenEmptyFolders()),
```

Comment:
Why do we need separate action for that? We can do it into `resetChosenPrompts`

Reply:
I think it's better to implement separate actions for improved separation of concerns, testability, and reusability.

Reply:
Code changed as suggested.